Source/WindowsServicify/WindowsServicify.Domain/ProcessManager.cs

```diff
@@ -40,7 +40,13 @@ private void StartProcess()
         process.StartInfo.FileName = _command;
         process.StartInfo.Arguments = _arguments;
         process.StartInfo.WorkingDirectory = _workingDirectory;
+        process.StartInfo.RedirectStandardOutput = true;
         process.Start();
+        while (!process.StandardOutput.EndOfStream)
+        {
+            string line = process.StandardOutput.ReadLine();
```

Comment:
There is an event based approach that might be better suited. This way you wont need an endless loop that might consume a lot of cpu without doing something useful.